app/swagger/schemas/forms.rb

```diff
@@ -38,14 +38,7 @@ class Forms
                 key :example, 'Information for Pre-Complaint Processing'
               end
 
-              property :first_issued_on do
-                key :description, 'The date the form first became available'
-                key :type, :string
-                key :example, '2012-01-01'
-                key :format, 'date'
-              end
-
-              property :last_revised_on do
+              property :last_revision_on do
```

Comment:
This was an inconsistency I caught between the Swagger docs and the actual response JSON.